client/src/components/Main/index.js

```diff
@@ -1,12 +1,17 @@
 import React from "react";
 import AppBar from "../Common/AppBar";
+import Map from "../Common/Map";
 import { eventPink } from "../../resources/style-constants";
 
 const Main = () => {
   return (
     <div>
       <AppBar color={eventPink} />
       Page coming soon ...
+
```

Comment:
The map component will be on the event viewer page and not the home page. Could you please revert your changes for this file?